### DISCUSSION TITLE
Unable to upload files to issue: POST https://github.com/upload/policies/assets 404

### DISCUSSION THREAD

**Discussion by DevDengChao** [community/community#31303] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/16363180/187345539-7fcd0bb7-d372-4762-8200-3e77eb06b567.png)

I was trying to upload a zip to https://github.com/mplushnikov/lombok-intellij-plugin/issues/1111#issue-1148968765, but this error shows up and the editor shows: `Something went really wrong, and we can’t process that file.`
![image](https://user-images.githubusercontent.com/16363180/187345991-826b618b-1d88-45db-a0c4-2f6efc3de0ba.png)

I've tried again today and found I was able to upload files in [the following comments in that issue](https://github.com/mplushnikov/lombok-intellij-plugin/issues/1111#issuecomment-1232413753) today, but I still can not find any documentation on attachment related limitations.

---

I've found a wrokaround: Upload the files in the new comment input textarea, then copy/paste the uploaded link into the desired comment.

**Comment by jfullstackdev** [comment]
how big is that file ? that is maybe the problem

**Reply by DevDengChao** [reply]
Almost 396kb. I can upload it here, but cannot upload it to the comment above.

[lombok-221.6008.13.zip](https://github.com/community/community/files/9454191/lombok-221.6008.13.zip)

**Comment by jfullstackdev** [comment]
how about using GitHub Desktop ? or drag and drop on the Web UI ?

**Reply by DevDengChao** [reply]
Never tried GitHub Desktop, drag and drop not works too.

**Reply by jfullstackdev** [reply]
I'm able to upload it without difficulty

![image](https://user-images.githubusercontent.com/47092464/187715338-ec780522-d757-4461-a1ae-1aca2dad8cbb.png)

![image](https://user-images.githubusercontent.com/47092464/187715490-db8b74c7-d568-45b7-a017-229eebab63a1.png)

maybe it's on your end, like your connection, local setup, maybe firewall or antivirus blocking it etc.

try logging in in Incognito too

**Reply by DevDengChao** [reply]
Not commiting files to repository, but attach many files to a single comment in issue.

**Reply by DevDengChao** [reply]
I tried uploading on both PC in my working place and home, none of these place works, and I can upload files in another comment in the same issue. I think it is not a connection problem.

**Comment by akphi** [comment]
Not 100% sure if this is related, but I got this error instead

```
attachment-upload.ts:114          POST https://github-production-user-asset-6210df.s3.amazonaws.com/ net::ERR_CERT_COMMON_NAME_INVALID
```

I could upload normally with `Firefox`

**Comment by ekfuhrmann** [comment]
I'm having this exact same issue. Using different browsers or even OS's is resulting in the same error. My files don't fail to upload 100% of the time, occasionally if I brute force it enough it will actually go through, but it's really unreliable.

@DevDengChao Did you ever figure out the problem on your end?

**Reply by DevDengChao** [reply]
No, still cannot upload more files to that comment. But I uploaded files in another comment in the same issue as a workaround.

**Reply by ekfuhrmann** [reply]
I was able to figure out the issue in my case. It turns out that in my case my ISP, CenturyLink, was blocking the file upload service due to a bad certification on their upload server (https://github-production-user-asset-6210df.s3.amazonaws.com//). I'm guessing that the reason uploads would occasionally work is because they were using a different AWS server or something.

![image](https://user-images.githubusercontent.com/8878152/207902409-2a8744f9-b5f4-474e-96f6-e476615844ea.png)

Once I told my ISP to permanently trust the upload service URL, I was then able to upload media again.

**Reply by jfullstackdev** [reply]
yup, told you that, mostly it's ISP blocking it and see that, it's the antivirus that is active there

**Reply by ekfuhrmann** [reply]
That antivirus is not local, but rather through the ISP actually. Really not a fan that they are intercepting my traffic like that, but once I whitelisted it we were good to go, so good guess on it being blocked. Hopefully this will help some other people that come across this problem in the future.

**Reply by jfullstackdev** [reply]
yup, it's not local, it's the ISP that is using it, so it blocks those that by default is not allowed

that's a common thing for others asking in this community as well, and I have been responding the same

**Comment by DevDengChao** [comment]
I'm able to upload files to the desired comment recently, so I'm going to close this discussion.
